### PR TITLE
Move identity-saml-sinatra from Github to Gitlab

### PR DESCRIPTION
**Pull request by lucasdze**
This PR contents all the code updates to move Circle-CI jobs to [Gitlab pipeline](https://gitlab.login.gov/lg/identity-saml-sinatra/-/pipelines/108836)

### CODE REVIEW

File: views/index.erb [hunk 1]
```diff
@@ -116,7 +116,7 @@
                       ['1', 'Authentication only (default)'],
                       ['2', 'Identity-verified'],
                       ['0', 'IALMax'],
-                      ['biometric-comparison-required', 'Biometric Comparison (Disabled in prod)'],
+                      ['biometric-comparison-required', 'Biometric Comparison'],
```
Comment (zachmargolis) [inline views/index.erb:119]: do we need to undo this change?

**Review by zachmargolis — APPROVED** [review]
LGTM once that template change gets addressd

**Review by dskgsa — APPROVED** [review]
